modules/mageck/count/meta.yml

```
@@ -0,0 +1,60 @@
+name: "mageck_count"
+description: mageck count for functional genomics, reads are usually mapped to a specific sgRNA
+keywords:
+  - sort
+  - functional genomics
+  - sgRNA
+  - CRISPR-Cas9
+tools:
+  - "mageck":
+      description: "MAGeCK (Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout), an algorithm to process, QC, analyze and visualize CRISPR screening data."
+      homepage: "https://sourceforge.net/p/mageck/wiki/Home/"
+      documentation: "https://sourceforge.net/p/mageck/wiki/demo/#step-4-run-the-mageck-count-command"
+      tool_dev_url: "None"
+      doi: ""
+      licence: "['BSD License']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - library:
+      type: file
+      description: library file containing the sgRNA and gene name
+      pattern: "*.{csv,txt,tsv}"
+  - reads:
```

This variable changed to `inptfile` now
```suggestion
  - inptfile:
```